actors/builtin/multisig/multisig_actor.go

```diff
@@ -78,18 +78,13 @@ func (a Actor) Constructor(rt vmr.Runtime, params *ConstructorParams) *adt.Empty
 		rt.Abortf(exitcode.ErrIllegalArgument, "must have at least one signer")
 	}
 
-	var signers []addr.Address
-	for _, sa := range params.Signers {
-		signers = append(signers, sa)
-	}
-
 	pending, err := adt.MakeEmptyMap(adt.AsStore(rt)).Root()
 	if err != nil {
 		rt.Abortf(exitcode.ErrIllegalState, "failed to create empty map: %v", err)
 	}
 
 	var st State
-	st.Signers = signers
+	st.Signers = params.Signers
```

Comment:
Did the linter catch this? That's pretty impressive if it did.

Reply:
It caught a simpler thing which reminded me to do this here.